pkg/cmd/openshift-controller-manager/controller/interfaces.go

```diff
@@ -207,6 +217,7 @@ func (c *ControllerContext) StartInformers(stopCh <-chan struct{}) {
 	c.TemplateInformers.Start(stopCh)
 	c.QuotaInformers.Start(stopCh)
 	c.RouteInformers.Start(stopCh)
+	c.SecurityInformers.Start(stopCh)
```

Comment:
why

Reply:
that was a previous attempt to fix things/understand what was going on, not needed, will set back.